h/features/models.py

```diff
@@ -105,3 +112,57 @@ def remove_old_flags(cls):
 
     def __repr__(self):
         return '<Feature {f.name} everyone={f.everyone}>'.format(f=self)
+
+
+class Cohort(Base, mixins.Timestamps):
```

Comment:
Okay, happy to keep this as `cohort` (and thus `Cohort`) for now, but please let's have `cohort_user`, not `user_cohort`.